go.mod

```
@@ -1,47 +1,29 @@
-module github.com/kiegroup/kogito-cloud-operator
+module github.com/kiegroup/rhpam-kogito-operator
+
+go 1.14
 
 require (
 	github.com/RHsyseng/operator-utils v0.0.0-20200304191317-2425bf382482
 	github.com/coreos/prometheus-operator v0.40.0
-	github.com/cucumber/gherkin-go/v11 v11.0.0
-	github.com/cucumber/godog v0.11.0
-	github.com/cucumber/messages-go/v10 v10.0.3
 	github.com/go-logr/logr v0.1.0
-	github.com/go-logr/zapr v0.1.1
 	github.com/go-openapi/spec v0.19.7
-	github.com/google/uuid v1.1.1
-	github.com/imdario/mergo v0.3.9
 	github.com/infinispan/infinispan-operator v0.0.0-20210106103300-03aa6d76d1b2
 	github.com/integr8ly/grafana-operator/v3 v3.4.0
 	github.com/keycloak/keycloak-operator v0.0.0-20200917060808-9858b19ca8bf
-	github.com/machinebox/graphql v0.2.2
-	github.com/matryer/is v1.4.0 // indirect
+	github.com/kiegroup/kogito-cloud-operator v0.0.3
```

```suggestion
	github.com/kiegroup/kogito-operator v1.4.0
```
After your renaming, right?

---

The tag that we create after renaming of `kogito-operator`, I will replace it here